src/EventListener/ExceptionListener.php

```diff
@@ -24,6 +25,23 @@
  */
 final class ExceptionListener extends BaseExceptionListener
 {
+    /**
+     * @var array
```

Comment:
Useless docblock, can be removed

Reply:
I agree with removing the block on the constructor, but the type infered for the property is still mixed, because it can be set from anywhere in the class (no `final` for properties like in java)

Reply:
I agree with dunglas